src/components/childComponents/step/CgeDescription.jsx

```diff
@@ -0,0 +1,89 @@
+import * as React from "react";
+import {
+	Box,
+	Table,
+	TableBody,
+	TableCell,
+	TableRow,
+	Typography,
+} from "@material-ui/core";
+import {makeStyles} from "@material-ui/core/styles";
+
+import {cgeInfo} from "../../../app.messages";
+
+const useStyles = makeStyles({
+	cgeDescriptionTitle: {
+		cursor: "pointer",
+		color: "#00619D",
+		fontFamily: "Rubik, sans-serif",
+		fontSize: "12px",
+		fontWeight: "500",
+		margin: "12px auto"
+	},
+	cgeDescriptionAbout: {
+		color: "#69707F",
+		fontFamily: "Rubik, sans-serif",
+		fontSize: "12px",
+	},
+	tableRowHead: {
+		color: "#69707F",
+		fontFamily: "Rubik, sans-serif",
+		fontSize: "12px",
+		fontWeight: "500",
+		textAlign: "center",
+		backgroundColor: "#EAEEFF",
+		border: "solid 2px #FFFFFF",
+		width: "30%"
+	},
+	tableRowCell: {
+		color: "#69707F",
+		fontFamily: "Rubik, sans-serif",
+		fontSize: "12px",
+		textAlign: "left",
+		border: "solid 2px #FFFFFF",
+		width: "70%"
+	}
+});
+
+export default function CgeDescription(props) {
+	const {cgeMetrics, ...others} = props;
+	const classes = useStyles();
+	let cgeDescription = cgeInfo[cgeMetrics];
+
+	let tableContent = (
+		<Table size="small" aria-label="cge description">
+			<TableBody>
+				{Object.keys(cgeDescription["rows"]).map((key) => (
+					<TableRow key={key} class>
```

Comment:
What's the purpose of the "class" keyword here? I think it can be removed

Reply:
typo... i'll remove them

Reply:
done